### DISCUSSION TITLE
Map moves away

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#987] [General]
Is there any way to lock the map to keep it away from disappearing when clicking anywhere on the screen?
Thank you

**Comment by wolfgang-ch** [comment]
Your map is propably synched

![grafik](https://user-images.githubusercontent.com/1283445/215960705-c9e97d68-2b80-422e-b926-2f924c964680.png)

**Reply by gringojack** [reply]
It doesnt behave any better Perhaps I do wrong
An other problem: How do I switch from A mode to B mode and reverse? (see attached image)
Thank you much
![MytourbookScreen](https://user-images.githubusercontent.com/4082252/216000357-e9d35222-fafd-4a17-938d-b2b8b20dbf94.jpg)

**Reply by wolfgang-ch** [reply]
> How do I switch from A mode to B mode and reverse? (see attached image) 

What do you want to switch?
